server/controllers/get-contract-by-id.js

```diff
@@ -38,8 +38,8 @@ module.exports = exports = async (req, res, next) => {
 			if (req.query.save !== '0') {
 				let contract_data = reformatSalesforceContract(JSON.parse(JSON.stringify(contract)));
 				contract_data.last_updated = new Date();
-				if (contract_data.orders) {
-					const activeOrder = contract_data.orders.find(order => order.status === 'Active');
+				if (contract.orders) {
```

Comment:
**Question**: Is `.orders` a new field that tells us it's a multiyear contract?

Reply:
yes.. added back in September I believe